### ISSUE TITLE
Aqara Camera G3 - Unable to update to latest FW

### ISSUE THREAD

**Issue by T-REX-XP** [niceboygithub/AqaraCameraHubfw#32]
**HW:**  lumi.camera.gwpagl01, model: AH_G3
**SW version:** 3.3.4_0007.0008
**How to reproduce**:
```
cd /tmp && wget -O /tmp/curl "http://master.dl.sourceforge.net/project/aqarahub/binutils/curl?viasf=1" && chmod a+x /tmp/curl
/tmp/curl -s -k -L -o /tmp/g3_update.sh https://raw.githubusercontent.com/niceboygithub/AqaraCameraHubfw/main/modified/G3/g3_update.sh
chmod a+x /tmp/g3_update.sh && /tmp/g3_update.sh
```
**Logs:**

```
ota progress=100
zigbee/send {"cmd":"report","id":2000000026,"did":"lumi.0","time":1706354846267,"rssi":0,"zseq":0,"params":[{"res_name":"8.0.2090","value":0}],"dev_src":"0"}
ota zigbee coordinator successful >>>>>>>>>>>>>>>
zigbee ota(ver: v1.0.0) firing up, built at Dec 16 2021 18:34:16; Author:
{"cmd":"read","id":7054,"did":"lumi.0","params":[{"res_name":"8.0.2080"}]}

zigbee/send {"cmd":"report","id":2000000027,"did":"lumi.0","time":1706354850438,"rssi":0,"zseq":0,"params":[{"res_name":"8.0.2080","value":"0008_2.06"}],"dev_src":"0"}
coor ver =0008
zigbee/send {"cmd":"report","id":2000000028,"did":"lumi.0","time":1706354850438,"rssi":0,"zseq":0,"params":[{"res_name":"8.0.2166","value":"{\"hostVer\":\"2.06\",\"coorVer\":\"0008\",\"networkUp\":1,\"panId\":\"0x7647\",\"txPower\":32,\"channel\":20}"}],"dev_src":"0"}
===Update uboot===
===Update kernel===
=== KERNEL1 ====
main:102
Erasing 128 Kibyte @ 4e0000 -- 100 % complete
/tmp/g3_update.sh: line 667: /bin/nandwrite: not found
/tmp/g3_update.sh: line 668: /bin/nanddump: not found
main:102
Erasing 128 Kibyte @ 4e0000 -- 100 % complete
/tmp/g3_update.sh: line 667: /bin/nandwrite: not found
/tmp/g3_update.sh: line 668: /bin/nanddump: not found
cat: can't open '/tmp/kernel_head': No such file or directory
main:102
Erasing 128 Kibyte @ 4e0000 -- 100 % complete
/tmp/g3_update.sh: line 667: /bin/nandwrite: not found
/tmp/g3_update.sh: line 668: /bin/nanddump: not found
cat: can't open '/tmp/kernel_head': No such file or directory
main:102
Erasing 128 Kibyte @ 4e0000 -- 100 % complete
/tmp/g3_update.sh: line 667: /bin/nandwrite: not found
/tmp/g3_update.sh: line 668: /bin/nanddump: not found
cat: can't open '/tmp/kernel_head': No such file or directory
Update failed, reason: OTA failed!
```

**Comment by niceboygithub** [comment]
Please upgrade 3.3.9 first. (remember that remember post_init.sh in /data/scripts to enable telnet)

**Comment by niceboygithub** [comment]
Please update to the latest g3_udpate.sh and try again

**Comment by sdavides** [comment]
with update 3.3.9 the problem is solved, but the reboot is back to 3.2

**Comment by T-REX-XP** [comment]
solved for me with latest updates.
